src/attributes/classes.js

```diff
@@ -7,16 +7,20 @@ define([
 
 var rclass = /[\t\r\n\f]/g;
 
+function getClass( elem ) {
+	return elem.getAttribute && elem.getAttribute( "class" ) || "";
```

Comment:
I realised I said the wrong, but you got what I meant. getAttribute can return null and this ensures we return a string.